node/src/neighborhood/node_location.rs

```diff
@@ -98,37 +91,25 @@ mod tests {
     }
 
     #[test]
-    fn create_gossip_node_with_addr_and_country_code_reveal_results_in_node_with_addr_adn_free_world_bit(
+    fn node_record_from_gossip_with_addr_and_country_is_populated_with_right_addr_and_free_world_bit(
```

Comment:
Sorry that I cannot comment on that directly, but I found out there is unneeded `        expected_node_record.resign();` in the test called `can_create_a_node_record_from_a_reference`. Please remove that line. The sign doesn't change from change in the metadata only.